server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/overview.ftl

```diff
@@ -24,7 +24,7 @@
         </div>
       </div>
       <div class="row">
-        <div class="col-sm-6" id="manager">
+        <div class="col-sm-6 col-sm-offset-3" id="manager">
```

Comment:
I think the information in this table can probably be "prettified" a bit, by moving it out of the HTML table format, and into some other format for presentation, but centering it is an acceptable interim solution.

Reply:
Hi @ctubbsii, If you have any thoughts/ideas on the format let me know I'm happy to work on it. Thanks

Reply:
I was thinking something like these boxes in this bootstrap example: https://getbootstrap.com/docs/4.0/examples/pricing/
But, I would defer to somebody more experienced with site design. This comment was mostly just a passing thought about how we could probably get rid of the table format for the information at the top of the overview page.